File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/triangle.py
from Quartz import CGPathCreateMutable,CGPathMoveToPoint, CGPathCloseSubpath, CGPathAddLineToPoint

name = "Triangle"

def create_cgpath(frame):
    newPath = CGPathCreateMutable()
    CGPathMoveToPoint(newPath, None, 0,0)
    CGPathAddLineToPoint(newPath, None, frame.size.width, 0)
    CGPathAddLineToPoint(newPath, None, frame.size.width/2, frame.size.height)
    CGPathCloseSubpath(newPath)
    return newPath

File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/triangle.js
var name = "Triangle"


var createPath = function(newPath, frame)
{
    newPath.moveToPoint({x:0, y:0});
    newPath.lineToPoint({x:frame.width, y:0});
    newPath.lineToPoint({x:frame.width/2, y:frame.height});
    newPath.closePath();
}
